src/gleam/float.gleam

```diff
@@ -115,3 +115,47 @@ pub external fn round(Float) -> Int =
 ///
 pub external fn truncate(Float) -> Int =
   "erlang" "trunc"
+
+/// Returns the absolute value of the input as a float.
+///
+/// ## Examples
+///
+///    > absolute_value(-12.5)
+///    12.5
+///
+///    > absolute_value(10.2)
+///    10.2
+///
+pub external fn absolute_value(Float) -> Float =
+  "erlang" "abs"
+
+/// Returns the results of the base being raised to the power of the
+/// exponent, as a float.
+///
+/// ## Examples
+///
+///    > power(2.0, 2.0)
+///    4.0
+///
+///    > power(8.0, 1.5)
+///    64.0
+///
+pub external fn power(base: Float, exponent: Float) -> Float =
+  "math" "pow"
+
+/// Returns the square root of the input as a float.
+///
+/// ## Examples
+///
+///    > square_root(4.0)
+///    2.0
```

Comment:
@cschembor3 should this not be `Ok(2.0)`

Reply:
ah you're totally right. Just opened https://github.com/gleam-lang/stdlib/pull/104 to address this. Thank you!